erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -249,7 +249,7 @@ def create_payment_entry(self, submit=True):
 		if (
 			party_account_currency == ref_doc.company_currency and party_account_currency != self.currency
 		):
-			party_amount = ref_doc.base_grand_total
+			party_amount = ref_doc.get("base_rounded_total") or ref_doc.get("base_grand_total")
```

Comment:
Shouldn't this apply to the other branch of the condition as well?

(`party_amount = self.get("rounded_total") or self.get("grand_total")`)

Probably not the issue you're trying to solve, but good for consistency and in accordance with other parts of the code.

Reply:
@barredterra  'Payment Request'  doctype doesn't have `rounded_total`.